lms/djangoapps/course_api/blocks/serializers.py

```diff
@@ -103,6 +103,7 @@ def __init__(
     "descendants",
     "authorization_denial_reason",
     "authorization_denial_message",
+    'contains_gated_content',
```

Comment:
This lets the `contains_gated_content` field appear in serializations of forbidden content (there's an allow-list for fields when the user doesn't have access).